EIPS/eip-5793.md

```diff
@@ -0,0 +1,52 @@
+
+
+---
+eip: 5793
+title: "eth/68: Add transaction type to tx announcement "
+description: "Adds the transaction type to tx announcement messages in the wire protocol"
+author: Marius van der Wijden (@MariusVanDerWijden)
+discussions-to: https://ethereum-magicians.org/t/eip-5793-eth-68-add-transaction-type-to-tx-announcement/11364
+status: Draft
+type: Standards Track
+category: Networking
+created: tbd
+requires: 2464, 2481, 4938
+---
+
+## Abstract
+
+The [Ethereum Wire Protocol](https://github.com/ethereum/devp2p/tree/master/caps/eth.md) defines request and response messages for exchanging data between clients. The `NewPooledTransactionHashesPacket` response announces transactions available in the node. We propose to extend the `NewPooledTransactionHashesPacket` such that the node sends both the transaction hashes as well as the transaction types as define in EIP-2718.
+
+## Motivation
+
+`NewPooledTransactionHashesPacket` was introduced to provide a fallback mechanism if transaction broadcast broke. The idea is for a node to announce which transactions it has locally to peers and let request the transactions. 
```

Comment:
That's not entirely correct. This message is not a fallback, it's just one part of the broadcast mechanism. You could just remove this sentence and state

```
The `NewPooledTransactionHashes` message announces transaction hashes, allowing the peer to selectively fetch transactions it does not yet have.
```